### PR TITLE
Add resource for LB cookie persistence profile

### PR DESCRIPTION
**Pull request by asarfaty**
the cookie_time field will be added in a followup patch

### CODE REVIEW

File: nsxt/resource_nsxt_lb_cookie_persistence_profile.go [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+				Optional:    true,
+				Default:     true,
+			},
+			"cookie_garble": &schema.Schema{
+				Type:        schema.TypeBool,
+				Description: "A boolean flag which reflects whether the cookie value (server IP and port) would be encrypted or in plain text",
+				Optional:    true,
+				Default:     true,
+			},
+			"cookie_domain": &schema.Schema{
```
Comment (annakhm) [inline nsxt/resource_nsxt_lb_cookie_persistence_profile.go]: I suggest to group all insert params into sub-struct

Comment (asarfaty) [reply nsxt/resource_nsxt_lb_cookie_persistence_profile.go]: ok. Done.

File: website/docs/r/lb_cookie_persistence_profile.html.markdown [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+* `cookie_fallback` - (Optional) A boolean flag which reflects whether once the server points by this cookie is down, a new server is selected, or the requests will be rejected.
+* `cookie_garble` - (Optional) A boolean flag which reflects whether the cookie value (server IP and port) would be encrypted or in plain text.
+* `insert_mode_params` - (Optional) Additional parameters for the INSERT cookie mode:
+  * `cookie_domain` - (Optional) HTTP cookie domain (for INSERT mode only).
+  * `cookie_path` - (Optional) HTTP cookie path (for INSERT mode only).
+  * `cookie_expiry_type` - (Optional) Type of cookie expiration timing (for INSERT mode only). Accepted values: SESSION_COOKIE_TIME for session cookie time setting and PERSISTENCE_COOKIE_TIME for persistence cookie time setting.
+
+        "max_idle_time": &schema.Schema{
```
Comment (annakhm) [inline website/docs/r/lb_cookie_persistence_profile.html.markdown]: Looks like this got here by mistake

Comment (asarfaty) [reply website/docs/r/lb_cookie_persistence_profile.html.markdown]: Thanks.

**Pull request merged**